dtkcore.cmake

```diff
@@ -91,9 +91,23 @@ install(FILES cmake/DtkTools/DtkSettingsToolsMacros.cmake
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/Dtk${DTK_VERSION_MAJOR}Tools"
     RENAME Dtk${DTK_VERSION_MAJOR}SettingsToolsMacros.cmake)
 
-install(FILES cmake/DtkDConfig/DtkDConfigConfig.cmake
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/Dtk${DTK_VERSION_MAJOR}DConfig"
-    RENAME Dtk${DTK_VERSION_MAJOR}DConfigConfig.cmake)
+if (DTK_VERSION_MAJOR STRLESS "6")
+    set(DEPRECATED_SINCE_DTK6 "
+# deprecated since dtk6
```

Comment:
可以用[=[]=]包裹起来防止转义，另外这个名字是不是可以体现dconfig,否则需要其它代码片段的话就麻烦了，
```
[=[
# deprecated since dtk6
function(dconfig_meta_files)
    dtk_add_config_meta_files(${ARGV})
endfunction()
function(dconfig_override_files)
    dtk_add_config_override_files(${ARGV})
endfunction()
]=]
```